README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/MediaMath/lambda-cron.svg?token=tMt81cZ8XUGin1RurU5s&branch=master)](https://travis-ci.com/MediaMath/lambda-cron)
```

Comment:
Does this token need to be manually invalided on travis-ci.com, since it is stored in previous commit history?